### DISCUSSION TITLE
Would love to be able to add Math into chart !

### DISCUSSION THREAD

**Discussion by gilhoo** [tone-row/flowchart-fun#296] [Features / Syntax]
Hi,
This would be great if mathematical equation through LaTeX could be used in FlowChart Fun.
Using MathJax or KaTeX

**Comment by rob-gordon** [comment]
@gilhoo This is a great idea and you are definitely not the first person to request it. It's on the roadmap. 

I'm currently working on a v2 that will support different text transformers, specifically like Markdown and ideally MathJax or KaTeX. Thank you for pointing me to those!

I'll update this discussion when that feature is ready. 👍
